### PR TITLE
New package: dreamchess-0.2.0

### PR DESCRIPTION
**Pull request by cr6git**
closes https://github.com/void-linux/void-packages/issues/1084

### CODE REVIEW

File: srcpkgs/dreamchess/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'dreamchess'
+pkgname=dreamchess
+version=0.2.1RC2
```
Comment (maxice8) [inline srcpkgs/dreamchess/template]: RC is Release Candidate which is a pre-release, use 0.2.0

**Comment by cr6git** [comment]
https://travis-ci.org/void-linux/void-packages/jobs/407805337#L919
````
configure: error: Cannot find Mini-XML library.
````

File: srcpkgs/dreamchess/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'dreamchess'
+pkgname=dreamchess
+version=0.2.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="flex"
+makedepends="SDL_image-devel SDL_mixer-devel glew-devel mxml-devel pugixml-devel"
+depends="desktop-file-utils hicolor-icon-theme"
+short_desc="3D OpenGL chess game"
+maintainer="Frank Steinborn <steinex@nognu.de>"
```
Comment (maxice8) [inline srcpkgs/dreamchess/template]: please assign yourself as maintainer

**Comment by steinex** [comment]
I just talked to dreamchess upstream if they would be willing to release a new stable version. They agreed and asked me to test a specific commit which they then will release as 0.2.2.

I will take care of it and I would also be ok with maintaining the package then.

OK @cr6git ? If you want you can close this PR and I will resubmit, or I'll ping you again so you can update the PR when 0.2.2. Whatever you prefer.

**Comment by cr6git** [comment]
@steinex I let you take care of it.

**Pull request closed without merging**